modules/workers/nodepools.tf

```diff
@@ -79,6 +79,11 @@ resource "oci_containerengine_node_pool" "workers" {
     lookup(each.value, "node_metadata", {}), # pool-specific
   )
 
+  node_eviction_node_pool_settings {
+    eviction_grace_duration              = format("PT%sM", each.value.eviction_grace_duration)
```

Comment:
the default value format is `PT%sS` , the consequence is that we always have a state change !

Reply:
Updated in https://github.com/oracle-terraform-modules/terraform-oci-oke/pull/791

Reply:
when  will this make it into `5.0.0-RC4` ?